custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssueNavigator.kt

```diff
@@ -1,41 +1,67 @@
-                        presenceOfElementLocated(By.id("issuetable")),
-                        presenceOfElementLocated(By.id("issue-content"))
-                    ),
-                    or(
-                        presenceOfElementLocated(By.id("jira-issue-header")),
-                        presenceOfElementLocated(By.id("key-val"))
-                    ),
-                    or(
-                        presenceOfElementLocated(By.id("new-issue-body-container")),
-                        presenceOfElementLocated(By.className("issue-body-content"))
-                    )
+        driver,
+        or(
+            and(
+                or(
+                    presenceOfElementLocated(By.cssSelector("ol.issue-list")),
+                    presenceOfElementLocated(By.id("issuetable")),
+                    presenceOfElementLocated(By.id("issue-content"))
                 ),
-                presenceOfElementLocated(By.className("no-results-hint")) // TODO is it too optimistic like in SearchServerFilter.waitForIssueNavigator ?
-            )
+                or(
+                    presenceOfElementLocated(By.id("jira-issue-header")),
+                    presenceOfElementLocated(By.id("key-val"))
+                ),
+                or(
+                    presenceOfElementLocated(By.id("new-issue-body-container")),
+                    presenceOfElementLocated(By.className("issue-body-content"))
+                )
+            ),
+            presenceOfElementLocated(By.className("no-results-hint")) // TODO is it too optimistic like in SearchServerFilter.waitForIssueNavigator ?
         )
+    )
         .cloudErrors()
         .timeout(Duration.ofSeconds(30))
         .build()
 
-    fun waitForNavigator() {
+    override fun waitForNavigator() {
         page.waitForPageToLoad()
     }
+
+    override fun selectIssue() {
+        val element = getIssueElementFromList()
+        val title = element.getAttribute("title")
+        element.click()
+        driver.wait(
+            ExpectedConditions.and(
+                ExpectedConditions.presenceOfElementLocated(By.xpath("//*[@data-test-id = 'issue.views.issue-base.foundation.summary.heading' and contains(text(), '$title')]")),
+                ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@aria-label='Not watching']")),
+                ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@data-test-id='issue.activity.comment']")),
+                ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@aria-label='Add attachment']"))
+            )
+        )
+    }
+
+    private fun getIssueElementFromList(): WebElement {
+        val elements = driver.wait(
+            ExpectedConditions.presenceOfAllElementsLocatedBy(By.xpath("//*[@class ='issue-list']/*[not(@class ='focused')]"))
+        )
+        val rndIndex = Random().nextInt(elements.size - 1)
+        return elements[rndIndex]
```

Comment:
We should use seeded randoms to get repeatable tests. So gotta pass the `SeededRandom` it from `Scenario` to `CloudIssueNavigator` via constructor and use it:
```kotlin
random.pick(elements)!!
```